big-int/src/bigint.hpp

```diff
@@ -40,110 +40,83 @@ class largeInt
     std::string number;
 
   public:
-    largeInt()
-    {
-    }
-
-    largeInt(int)
-    {
-    }
-    largeInt(int64_t)
-    {
-    }
-    largeInt(long long)
-    {
-    }
-    largeInt(std::string)
-    {
-    }
+    largeInt();
+
+    largeInt(int);
+    largeInt(long long int);
```

Comment:
Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285290)